### ISSUE TITLE
Make it easier for users to do batch revisions

### ISSUE THREAD

**Issue by anna-parker** [loculus-project/loculus#5296]
We don't have an easy way for users to do batch revisions. This means they need to have their old original metadata which they need to add loculus accessions to (and revise) or they download the metadata from us with the loculus accessions. The issue with the second option (although it is probably easiest for submitters) users download the processed data and not the original data. For example we reformat authors lists from:
`Tiangay P.M.S Kallom` to `Tiangay P. M. S. Kallom`. 

No change is seen on the page but the ena deposition pod flagged this as a change in author's lists and wanted us to do a manual revision.

An easy solution is to change the ena deposition pod to compare the final authors output and not the original data in this case but in general this is an issue that will probably present itself in other ways.

https://loculus.slack.com/archives/C05G172HL6L/p1761121808948159
